bigtable/filter_samples.rb

```diff
@@ -22,147 +20,147 @@
   # [END bigtable_filters_limit_row_sample]
 end
 
-def filter_limit_row_regex project_id, instance_id, table_id
+def filter_limit_row_regex instance_id, table_id
   # [START bigtable_filters_limit_row_regex]
   filter = Google::Cloud::Bigtable::RowFilter.key ".*#20190501$"
-  read_with_filter project_id, instance_id, table_id, filter
+  read_with_filter instance_id, table_id, filter
   # [END bigtable_filters_limit_row_regex]
 end
 
-def filter_limit_cells_per_col project_id, instance_id, table_id
+def filter_limit_cells_per_col instance_id, table_id
   # [START bigtable_filters_limit_cells_per_col]
   filter = Google::Cloud::Bigtable::RowFilter.cells_per_column 2
-  read_with_filter project_id, instance_id, table_id, filter
+  read_with_filter instance_id, table_id, filter
   # [END bigtable_filters_limit_cells_per_col]
 end
 
-def filter_limit_cells_per_row project_id, instance_id, table_id
+def filter_limit_cells_per_row instance_id, table_id
   # [START bigtable_filters_limit_cells_per_row]
   filter = Google::Cloud::Bigtable::RowFilter.cells_per_row 2
-  read_with_filter project_id, instance_id, table_id, filter
+  read_with_filter instance_id, table_id, filter
   # [END bigtable_filters_limit_cells_per_row]
 end
 
-def filter_limit_cells_per_row_offset project_id, instance_id, table_id
+def filter_limit_cells_per_row_offset instance_id, table_id
   # [START bigtable_filters_limit_cells_per_row_offset]
   filter = Google::Cloud::Bigtable::RowFilter.cells_per_row_offset 2
-  read_with_filter project_id, instance_id, table_id, filter
+  read_with_filter instance_id, table_id, filter
   # [END bigtable_filters_limit_cells_per_row_offset]
 end
 
-def filter_limit_col_family_regex project_id, instance_id, table_id
+def filter_limit_col_family_regex instance_id, table_id
   # [START bigtable_filters_limit_col_family_regex]
   filter = Google::Cloud::Bigtable::RowFilter.family "stats_.*$"
-  read_with_filter project_id, instance_id, table_id, filter
+  read_with_filter instance_id, table_id, filter
   # [END bigtable_filters_limit_col_family_regex]
 end
 
-def filter_limit_col_qualifier_regex project_id, instance_id, table_id
+def filter_limit_col_qualifier_regex instance_id, table_id
   # [START bigtable_filters_limit_col_qualifier_regex]
   filter = Google::Cloud::Bigtable::RowFilter.qualifier "connected_.*$"
-  read_with_filter project_id, instance_id, table_id, filter
+  read_with_filter instance_id, table_id, filter
   # [END bigtable_filters_limit_col_qualifier_regex]
 end
 
-def filter_limit_col_range project_id, instance_id, table_id
+def filter_limit_col_range instance_id, table_id
   # [START bigtable_filters_limit_col_range]
   range = Google::Cloud::Bigtable::ColumnRange.new("cell_plan").from("data_plan_01gb").to("data_plan_10gb")
   filter = Google::Cloud::Bigtable::RowFilter.column_range range
-  read_with_filter project_id, instance_id, table_id, filter
+  read_with_filter instance_id, table_id, filter
```

Comment:
Could you add comments describing the variables that are set outside the region tags, similar to https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/8ef68b062146ae22eeaac8671bc0ac1bdacba3ad/servicedirectory/servicedirectory.rb#L17

Reply:
For storage samples, @JesseLovelace [commented](https://github.com/googleapis/google-cloud-ruby/pull/7823#pullrequestreview-513136492) that a combination of description and example value is preferred. Would you agree for Bigtable?

```
# Your Google Cloud project ID
# project = "your-project-id"
```

Reply:
The snippet rubric doesn't require the explanation. I'd say add a description if you feel one is needed, otherwise an example value is fine.

Reply:
OK, I'll just add example values for now:

```
# project = "your-project-id"
```

Reply:
@TheRoyalTnetennba you wrote:

> Could you add comments describing the variables that are set outside the region tags

But did you mean *inside* the region tags?

Reply:
@TheRoyalTnetennba I added comments describing the variables wherever possible. In some region tags, the `table` is implicit (the lookup is not shown), so in those I did not add the comments.